CHANGELOG.md

```diff
@@ -1 +1,14 @@
 # Changelog
+
+## [0.0.3] - 2019-06-02
+
+### Added 
+* Added variables to define strings used in `_volcano.py` graph labels.  
```

Comment:
I would suggest writing at the imperative, like we do in commit messages (in a spirit of standardization, e.g., https://github.com/conventional-changelog/conventional-changelog).

Reply:
@mkcor A bit conflicted about this, since other Dash libraries (like Dash itself) use past tense (https://github.com/plotly/dash/blob/master/CHANGELOG.md). It also makes a bit more sense to me personally to have commit messages in present tense, since they have yet to be merged, and to have changelog messages in past tense, since those changes have already been implemented and released.